src/App.vue

```diff
@@ -768,6 +768,10 @@ export default {
 body .modal-wrapper * {
 	box-sizing: border-box;
 }
+
+.modal-wrapper h2 {
```

Comment:
The rule is too wide, but we don't seem to suffer from it, so OK for now

Reply:
Yes, as the issue is also wide :p